pkg/ebpf/c/tracee.bpf.c

```diff
@@ -83,6 +83,7 @@
 #define FILE_MAGIC_HDR_SIZE 32        // magic_write: bytes to save from a file's header
 #define FILE_MAGIC_MASK     31        // magic_write: mask used for verifier boundaries
 #define NET_SEQ_OPS_SIZE    4         // print_net_seq_ops: struct size
+#define NET_SEQ_OPS_TYPES   6         // print_net_seq_ops: argument size
```

Comment:
This can be passed as an argument to the ioctl, which also has the advantage of removing the need to sync it with userspace struct size.

Reply:
Can we do this in a different PR? this will take some testing

Reply:
Mind opening an issue and putting a TODO in the code so we don't forget ?